packages/core/lib/testing/testrunner.js

```diff
@@ -175,7 +175,11 @@ TestRunner.prototype.endTest = async function(mocha) {
     fromBlock: this.currentTestStartBlock.toNumber()
   });
 
-  if (logs.length === 0) {
+  const filteredLogs = logs.filter(log => {
```

Comment:
Nitpick: we could name this more meaningfully than `filteredLogs`. Maybe `userDefinedEvents`?

Reply:
Sure, that seems like a good idea!